all/projectAlcazar/src/sys/max7219.c

```diff
@@ -78,6 +86,25 @@ void putChar(char ch, unsigned char pos)
 	}
 }
 
+//important: set decode mode to eDecode_allCharactersCustomFont before using this function
+//displays string stringDisplayed with length of stringLen
+void putString(char stringDisplayed[], unsigned int stringLen)
+{
+	int i, j, string_pos;
+	for (string_pos = 0; string_pos < stringLen; string_pos++) // for a whole pass of the string
+	{
+		busyDelayMs(750); //delay before sending each char
+		putChar(MAX7219_font[stringDisplayed[string_pos]-32], 0); //send each char to first position of display
```

Comment:
32 can also be written as ' ' for better undersranding, just a suggestion

Reply:
Indeed, thank you!